src/util/modules.py

```diff
@@ -540,6 +540,18 @@ def _up(table, old, new):
         force_install_module(cr, into, reason="installed {!r} module has been merged into it".format(old))
 
 
+def _force_install_new_dependency(cr, module, reason="it is a new dependency of an installed module"):
```

Comment:
Why not inlining this right in the only usage we have for it? Everything is clearer in that context.

Reply:
moved